beancount_reds_importers/libreader/csvreader.py

```diff
@@ -108,10 +108,17 @@ def convert_columns(self, rdr):
         # fixup currencies
         def remove_non_numeric(x):
             return re.sub("[^0-9\.-]", "", str(x).strip())  # noqa: W605
+
+        def add_decimal(x):
+            if '.' not in x:
+                return x+".00"
```

Comment:
You may also consider having `add_currency_precision` be `False` or an integer, which would specify the number of zeros to add.